common/display/displayqueue.h

```diff
@@ -54,6 +54,7 @@ class DisplayQueue {
 
   bool Initialize(uint32_t pipe, uint32_t width, uint32_t height,
                   DisplayPlaneHandler* plane_manager);
+  bool Exit(uint32_t power_mode);
```

Comment:
No need of power_mode, probably make it more descriptive of what it means when this is called. HandleDisconnect ?

Reply:
do